README.md

```diff
@@ -61,6 +61,25 @@ The release process is building all runtimes and then puts them into a release i
 
 The format of [`CHANGELOG.md`](CHANGELOG.md) is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## Release guidelines
+
+Here are the recommended steps for both major and minor runtime releases.
+
+|Steps |Description |Major |Minor |
```

Comment:
We actually only do minor releases all the time :see_no_evil:

Reply:
Maybe just mention that we don't really follow semver? We are doing minor bumps for `polkadot-sdk` bumps and otherwise we only use the `patch` version for other changes.

Reply:
> We actually only do minor releases all the time 🙈

For runtime releases? What I noted is that [v1.2.0](https://github.com/polkadot-fellows/runtimes/releases/tag/v1.2.0) = Major and [v1.2.8](https://github.com/polkadot-fellows/runtimes/releases/tag/v1.2.8) = minor; etc.
Please correct me if I am wrong about this rationale.

Reply:
> Maybe just mention that we don't really follow semver? We are doing minor bumps for `polkadot-sdk` bumps and otherwise we only use the `patch` version for other changes.

As Eskimor suggested, I will provide a link to the [docs for polkadot-sdk](https://github.com/paritytech/polkadot-sdk?tab=readme-ov-file#-releases), rather than add too many granular details to this page.

Reply:
MAJOR.MINOR.PATCH

This is how the individual numbers of the version number are called. So, `1.2.0` is a minor release.

Reply:
> MAJOR.MINOR.PATCH

Ok, got it.
I have edited the doc. Thanks!